### PR TITLE
Use Selenium Standalone instead of Webdrivers

### PR DESCRIPTION
**Pull request by tpendragon**
I couldn't get selenium/webdrivers working locally, and this takes local environment out of the equation.

### CODE REVIEW

**Review by eliotjordan — CHANGES_REQUESTED** [review]
I found one small issue in the lando config, otherwsie this looks great. I'm 👍 on merging this.

File: .lando.yml [hunk 1]
```diff
@@ -18,6 +18,25 @@ services:
...
     portforward: true
+  chrome:
+    type: compose
+    app_mount: false
+    services:
+      image: seleniarm/standalone-chromium:114.0
+      environment:
+        START_XVFB: "false"
+        SE_NODE_MAX_SESSIONS: 20
+        SE_NODE_OVERRIDE_MAX_SESSIONS: true
```
[the remark this answers is not in the document]
Comment (tpendragon) [reply .lando.yml]: Fixed, thank you!

**Review by eliotjordan — APPROVED** [review]

**Comment by hackartisan** [comment]
I ran on the spec/features directory and got a lot of `Net::ReadTimeout with #<Socket:(closed)>` -- maybe corresponding to the specs marked `js: true`?

**Comment by tpendragon** [comment]
> I ran on the spec/features directory and got a lot of `Net::ReadTimeout with #<Socket:(closed)>` -- maybe corresponding to the specs marked `js: true`?

Interesting! Maybe we can pair at some point and look at it, these pass for me.

**Comment by hackartisan** [comment]
ran the tests in parallel. I had a handful of failures like this 

```
    1.1) Failure/Error: visit user_cas_omniauth_authorize_path                                                                                                                          
                                                                                                                                                                                         
          Selenium::WebDriver::Error::ServerError:                                                                                                                                       
            status code 404                                                                                                                                                              
          # ./spec/support/features/session_helpers.rb:13:in `sign_in'                                                                                                                             # ./spec/features/numismatic_reference_spec.rb:11:in `block (2 levels) in <top (required)>'    
```

They all passed when I ran a follow-up `rspec --only-failures`

a few other failures I have related to cogeo-mosaic seem unrelated to this PR.

I'm not sure I consider this blocking at this point. maybe it's good enough.

**Comment by hackartisan** [comment]
I ran in parallel again and didn't have any of those 404 failures.

**Pull request merged**